src/functions/keycloak/update-user/handler.ts

```diff
@@ -0,0 +1,47 @@
+import type { APIGatewayProxyEvent, APIGatewayProxyHandler } from "aws-lambda";
+import { CreateKeycloakApiService } from "src/database/services/keycloak-api-service";
+import { middyfy } from "src/libs/lambda";
+
+/**
+ * Lambda handler to update a user's attributes
+ * 
+ * @param event API Gateway event 
+ * @returns Response message as JSON string
+ */
+const updateUserAttributeHandler: APIGatewayProxyHandler = async (
+  event: APIGatewayProxyEvent
+) => {
+  try {
+    if (!event.body) {
+      throw new Error("Request body is missing.");
+    }
+
+    const body = JSON.parse(JSON.stringify(event.body));
+    const { id, attributes } = body;
+
+    if (!id || !attributes || typeof attributes !== "object") {
+      throw new Error("Missing or invalid body parameters: id, attributes.");
+    }
+
+    const api = CreateKeycloakApiService();
+
+    if (!attributes.isActive) {
+      attributes.isActive = ["Active"];
+    }
```

Comment:
I'm not sure about this, shouldn't the attribute name be severaId and shouldn't the value be the severaId?
As such this lambda also seems to be missing the functionality to fetch the user from Severa to obtain the id and add it as an attribute. Perhaps the lambda can be actually be provided the severaId as part of the attribute.

I'm thinking when the user clicks the checkbox to grant permission in the UI, first there is a call to severa to update the users keyword, then the lambda returns the user Id, which is passed to this lambda call for updating the keycloak user. It might be also reasonable to combine all of this functionality into a single lambda making both severa and keycloak requests rather than splitting it apart.

In fact I think combining the functionality in these lambdas is the way to go, but let me know what you think is best. Also something similar needs to be done for the remove e.g. not just remove the attribute from keycloak but also removing the keyword from the severa user.

It might be worth checking with Ilmo incase anyone else has started onthe severa Keyword functionality but I'm not aware of it,